docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml

```diff
@@ -589,33 +597,211 @@ depending on where they wind up.-->
+          package map parts. Open the package map to see the package parts
+          tab.</para>
+
+          <para><figure>
+              <title>Package Parts</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWAPkgParts01.jpg"
+                             vendor="ECLWatchSS" />
+                </imageobject>
+              </mediaobject>
+            </figure></para>
+
+          <para>You can examine the individual parts, add parts, or remove
+          parts through this interface in ECLWatch.</para>
+
+          <para><emphasis role="bold">Add Part</emphasis></para>
+
+          <para>To add a part to the package map, select the <emphasis
+          role="bold">Package Parts</emphasis> tab. Then press the <emphasis
+          role="bold">Add</emphasis> button. Fill in the appropriate
+          information then press <emphasis
+          role="bold">Apply</emphasis>.</para>
+
+          <para><emphasis role="bold">Remove Part</emphasis></para>
+
+          <para>To remove a part from the package map, select the <emphasis
+          role="bold">Package Parts</emphasis> tab. Then press the <emphasis
+          role="bold">Remove</emphasis> button and confirm that you really
+          want to remove the part.</para>
+
+          <para><emphasis role="bold">Get Part </emphasis><figure>
+              <title>Get Part</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWaGetPart01.jpg"
+                             vendor="ECLWatchSS" />
+                </imageobject>
+              </mediaobject>
+            </figure></para>
+
+          <para>Press the <emphasis role="bold">Get Part</emphasis> button to
+          view the contents of the selected part. <figure>
+              <title>Package Part Contents</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWaGetPart02.jpg"
+                             vendor="ECLWatchSS" />
+                </imageobject>
+              </mediaobject>
+            </figure></para>
+        </sect3>
+
+        <sect3>
+          <title id="Packagemap_Validation">Package Map Validation</title>
+
+          <para>To validate your package map:</para>
+
+          <para><figure>
+              <title>Validate Package Maps</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWaPkMgr.jpg" />
+                </imageobject>
+              </mediaobject>
+            </figure></para>
 
-        <para>Choose the <emphasis role="bold">Target</emphasis> and <emphasis
-        role="bold">Process</emphasis> from the drop lists on the Validate
-        Package Map tab.</para>
+          <orderedlist>
+            <listitem>
+              <para>Select the <emphasis role="bold">Validate Package
```

Comment:
The validate Package Map tab is for validating Active packagemaps or packagemaps that are not yet loaded (Validate Package Content)... 

you can also validate any packagemap on the system by:

PackageMapsTab --> select PackageMap  -> Click Open --> click on validate tab.
This will open a tab with the results of validating that specific packagemap.  Some of these packagemaps are not active and therefore could not be validated via the "validate Package Map tab" above.